### PR TITLE
fix error

### PR DESCRIPTION
**Pull request by hunter007**
因为删除的是单个页面，方法名称改为delete_page()，这点会造成不兼容之前版本。

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request
- Commit: f50c2cfbb0d7051dbbc35aa48c369e710293c00b
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/jxtech/wechatpy/blob/master/CONTRIBUTING.md

---

This message was auto-generated by https://gitcop.com

File: wechatpy/client/api/shakearound.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ def update_page(self, page_id, title, description,
         )
         return res['data']
 
-    def search_pages(self, page_ids, begin=0, count=10):
+    def search_pages(self, page_ids=[], begin=0, count=10):
```
Comment (messense) [inline wechatpy/client/api/shakearound.py]: 不要使用可变默认参数，应该改为：

``` python
def search_pages(self, page_ids=None, begin=0, count=10):
    page_ids = page_ids or []
```

**Comment by GitCop** [comment]
There were the following issues with your Pull Request
- Commit: f50c2cfbb0d7051dbbc35aa48c369e710293c00b
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 1847faef903500c28ea71ef47ae45717d2977f35
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/jxtech/wechatpy/blob/master/CONTRIBUTING.md

---

This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request
- Commit: f50c2cfbb0d7051dbbc35aa48c369e710293c00b
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 1847faef903500c28ea71ef47ae45717d2977f35
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: f3069d6d4ff84d27deecfbeba36867f0badf2312
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/jxtech/wechatpy/blob/master/CONTRIBUTING.md

---

This message was auto-generated by https://gitcop.com

**Comment by messense** [comment]
Thanks.

**Pull request merged**